src/components/Sidebar/Sidebar.js

```diff
@@ -1,10 +1,12 @@
 import React from "react";
 import styles from "./Sidebar.scss";
 
-const Sidebar = () => (
-  <div className={styles.container}>
-    <p>Sidebar</p>
-  </div>
-);
+const Sidebar = () => {
+  return (
+    <div className={styles.SideBar}>
```

Comment:
if we decided to use classname the same as a component name, shouldn't it be {styles.Sidebar} ?

Reply:
Yup, thanks!